common_test.go

```diff
@@ -0,0 +1,27 @@
+package common
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/zeebo/errs"
+)
+
+var statusFromErrorTests = []struct {
```

Comment:
i'd consider moving this into the TestStatusFromErrors test itself... it's unlikely to be used elsewhere